modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of AWS Load Balancer Controller
+
+After installing the operator, you can create an instance of `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The operator.only reconciles the resource with the name `cluster`.
+
+.Prerequisites
+
+.Procedure
+
+. Create the `AWSLoadBalancerController` resource:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController <1>
```

Comment:
This is not the resource group.

Reply:
Should we remove the <1> callout or should we keep it and change the description for that callout. For now, I kept the callout and changed its description.